index.html

```diff
@@ -11,6 +11,17 @@
 
     <header class="app-header">
         <a href="#" class="logo">Narson</a>
+
+        
+        <div id="myNav" class="overlay">
+            <a href="javascript:void(0)" class="closebtn">&times;</a>
+            <div class="overlay-content">
+                <a id="portfolio" href="#work">Portfolio</a>
+                <a id="about" href="#about">About</a>
+                <a id="contact" href="#">Contact</a>
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), when the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

Kindly update the `href` of the anchor tag in the `Contact` so that when the user clicks on it, the correct part of the page is displayed. 💪🛠